exchanges/bitstamp/bitstamp_test.go

```diff
@@ -140,28 +154,45 @@ func TestGetFee(t *testing.T) {
 	}
 }
 
-func TestCalculateTradingFee(t *testing.T) {
+func TestGetAccountTradingFee(t *testing.T) {
 	t.Parallel()
 
-	newBalance := make(Balances)
-	newBalance["BTC"] = Balance{
-		USDFee: 1,
-		EURFee: 0,
+	if !mockTests {
+		sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
 	}
 
-	if resp := b.CalculateTradingFee(currency.BTC, currency.USD, 0, 0, newBalance); resp != 0 {
-		t.Error("GetFee() error")
-	}
-	if resp := b.CalculateTradingFee(currency.BTC, currency.USD, 2, 2, newBalance); resp != float64(4) {
-		t.Errorf("GetFee() error. Expected: %f, Received: %f", float64(4), resp)
+	fee, err := b.GetAccountTradingFee(context.Background(), currency.NewPair(currency.LTC, currency.BTC))
+	if assert.NoError(t, err, "GetAccountTradingFee should not error") {
+		if mockTests {
+			assert.Positive(t, fee.Fees.Maker, "Maker should be positive")
+			assert.Positive(t, fee.Fees.Taker, "Taker should be positive")
+		}
+		assert.NotEmpty(t, fee.Symbol, "Symbol should not be empty")
+		assert.Equal(t, fee.Symbol, "ltcbtc", "Symbol should be correct")
 	}
-	if resp := b.CalculateTradingFee(currency.BTC, currency.EUR, 2, 2, newBalance); resp != float64(0) {
-		t.Errorf("GetFee() error. Expected: %f, Received: %f", float64(0), resp)
+
+	_, err = b.GetAccountTradingFee(context.Background(), currency.EMPTYPAIR)
+	assert.ErrorIs(t, err, currency.ErrCurrencyPairEmpty, "Should get back the right error")
+}
+
+func TestGetAccountTradingFees(t *testing.T) {
+	t.Parallel()
+
+	if !mockTests {
+		sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
 	}
 
-	dummy1, dummy2 := currency.NewCode(""), currency.NewCode("")
-	if resp := b.CalculateTradingFee(dummy1, dummy2, 0, 0, newBalance); resp != 0 {
-		t.Error("GetFee() error")
+	fees, err := b.GetAccountTradingFees(context.Background())
+	if assert.NoError(t, err, "GetAccountTradingFee should not error") {
+
```

Comment:
The linter is complaining about this empty line, please RM and the test should pass:

https://ci.appveyor.com/project/thrasher-/gocryptotrader-vxsws/builds/48230331#L299

Reply:
done, thank you :)